test/cypress/support/index.js

```diff
@@ -0,0 +1,23 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+import './commands'
```

Comment:
Always add the extension as part of the import (for clarity and for future compatibility with #586)

Reply:
Sure. To avoid future "picky" comments like this one I've added the rule to our eslint config so it automatically detects the error.